constantine/serialization/codecs_banderwagon.nim

```diff
@@ -30,10 +28,6 @@ import
   ../math/io/[io_bigints, io_fields],
   ./codecs_status_codes
 
-type
-  EC_Prj* = ECP_TwEdwards_Prj[Fp[Banderwagon]]
-  EC_Aff* = ECP_TwEdwards_Aff[Fp[Banderwagon]]
```

Comment:
if using multiple curves like BN254, BLS12_381, and Banderwagon, those shortnames are confusing. They should be private or not used.